app/controllers/api/api_controller.rb

```diff
@@ -17,6 +17,10 @@ class ApiController < ActionController::Base
       render_error 404, 'Not found'
     end
 
+    rescue_from ActiveRecord::RecordInvalid do |e|
+      render_error 422, e.record.errors.full_messages
```

Comment:
I just got notified about this message, 5 hours later. Good point, better to have them consistent.